package.json

```diff
@@ -1,25 +1,32 @@
 {
   "name": "@launchpadlab/eslint-config",
-  "version": "2.7.0",
+  "version": "3.0.0",
   "main": "index.js",
   "license": "MIT",
   "author": "dpikt",
+  "engines": {
+    "node": ">= 10"
+  },
   "scripts": {
     "test": "jest"
   },
   "dependencies": {
-    "babel-eslint": "^8.0.2",
-    "eslint-import-resolver-webpack": "^0.8.3",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-react": "^7.10.0"
+    "@babel/core": ">= 7.2.0",
+    "@babel/eslint-parser": "^7.21.3",
+    "eslint": "^8.0.0",
+    "eslint-import-resolver-webpack": "^0.13.2",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-react": "^7.32.2",
+    "eslint-plugin-react-hooks": "^4.6.0"
   },
   "devDependencies": {
-    "eslint": "^5.0.0",
-    "jest": "^21.2.1",
+    "@launchpadlab/babel-preset": "^2.1.1",
+    "jest": "^29.5.0",
     "react": "^16.8.6",
-    "webpack": "^3.8.1"
+    "webpack": "^5.0.0"
   },
-  "peerDependencies": {
-    "eslint": ">= 3.0.0"
+  "resolutions": {
+    "babel-plugin-lodash/@babel/types": "~7.20.0"
```

Comment:
This plugin is kind of abandonware it seems. It relies on a deprecated method and pollutes the console with a bunch of messages about it. Since it's just a dev dependency, I really don't care about pinning it